MyScript/index.js

```diff
@@ -0,0 +1,67 @@
+//const inputDate = '11/27/2021'
+const inputDate = prompt('Enter the date You are interested in: ', '11/26/2021')
+const inputDateFull = new Date(inputDate)
+
+const checkEnteredDay = (enteredDate) => {
+    const twoDayMs = 86400000 * 2
+    const firstWorkDayMs = new Date(2021, 10, 26).getTime()
+
+    const inputDateMs = new Date(enteredDate).getTime()
+    const differenceMs = inputDateMs - firstWorkDayMs
```

Comment:
```differenceMs``` and ```divisionResult``` don't need their own variables, as the same understanding of the variable can be get just from looking at the ```inputDateMs - firstWorkDayMs``` operation